libportability-gfx/src/conv.rs

```diff
@@ -341,6 +341,18 @@ pub fn map_stage_flags(stages: VkShaderStageFlags) -> pso::ShaderStageFlags {
     flags
 }
 
+pub fn map_pipeline_stage_flags(stages: VkPipelineStageFlags) -> pso::PipelineStage {
+    let max_flag = VkPipelineStageFlagBits::VK_PIPELINE_STAGE_HOST_BIT as u32;
+
+    if (stages & 1 << (max_flag + 1) - 1) == 0 {
+        // HAL flags have the same numeric representation as Vulkan flags
+        unsafe { mem::transmute(stages) }
```

Comment:
🎉 
I think we should be using more concrete things instead of `transmute`, e.g. `pso::PipelineStage::from_bits`

Reply:
I'll keep that it mind! Need a separate pass of aligning all flags and enum variants in gfx.